core-tests/shared/src/test/scala/zio/ZIOAppSpec.scala

```diff
@@ -49,7 +49,7 @@ object ZIOAppSpec extends ZIOBaseSpec {
         }
       }
 
-      val app1 = ZIOAppDefault(ZIO.fail("Uh oh!"), RuntimeConfigAspect.addLogger(logger1))
+      val app1 = ZIOApp(ZIO.fail("Uh oh!"), ZLayer.addLogger(logger1))
```

Comment:
Looking at how this reads, I question whether `ZLayer` is the right place for these things.

Yes, they are ZLayer constructors, but they are connected to the concept of runtime system / bootstrapping, not really part of the ZLayer domain (derived and domain-specific versus primitive constructors).

I don't really have a good suggestion but I could imagine creating an `object` like `BootstrapLayers` or `RuntimeLayers` to contain all of them.

Reply:
I agree and had the same thought. Another idea would be perhaps they could be moved to the `Runtime` companion object.